easybuild/easyconfigs/r/Ray-project/Ray-project-0.8.4-foss-2019b-Python-3.7.4.eb

```diff
@@ -3,7 +3,7 @@
 
 easyblock = 'PythonBundle'
 
-name = 'Ray'
+name = 'Ray-project'
```

Comment:
please adapt `source_tmpl` (below) to account for this change.

Reply:
Ah, thanks for the notice :)